src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -1941,9 +1941,9 @@ def test_cosmosdb_sql_role(self, resource_group):
             self.check('principalId', principal_id)
         ])
 
-        assert self.cmd('az cosmosdb sql role assignment exists -g {rg} -a {acc} -i {role_assignment_id3}').get_output_in_json() == False
+        assert self.cmd('az cosmosdb sql role assignment exists -g {rg} -a {acc} -i {role_assignment_id3}').get_output_in_json() is False
```

Comment:
```suggestion
        assert not self.cmd('az cosmosdb sql role assignment exists -g {rg} -a {acc} -i {role_assignment_id3}').get_output_in_json()
```

Reply:
I think `is False` is more precise.

These all return True:

```python
In [0]: assert not 0

In [1]: assert not None

In [2]: assert not False

In [3]: assert not ""
```